From 6e80c60b89728de28267242f7373ecf553e40bc1 Mon Sep 17 00:00:00 2001
From: Daniel P. Berrange <berrange@redhat.com>
Date: Thu, 28 May 2009 13:15:57 +0000
Subject: [PATCH] Avoid broken networking with new QEMU/KVM >= 86

(cherry picked from commit 2afc3bfd8b779ddba974da9d66d6ea337fc91c01)

Fedora-patch: libvirt-0.6.2-avoid-broken-networking-with-newer-qemu.patch
---
 src/qemu_conf.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/qemu_conf.c b/src/qemu_conf.c
index fc0e772..99f13c6 100644
--- a/src/qemu_conf.c
+++ b/src/qemu_conf.c
@@ -658,8 +658,8 @@ qemudNetworkIfaceConnect(virConnectPtr conn,
     }
 
     snprintf(tapfdstr, sizeof(tapfdstr),
-             "tap,fd=%d,script=,vlan=%d,ifname=%s",
-             tapfd, vlan, net->ifname);
+             "tap,fd=%d,vlan=%d",
+             tapfd, vlan);
 
     if (!(retval = strdup(tapfdstr)))
         goto no_memory;
-- 
1.6.2.5

